### PR TITLE
Adds ui to diary app

### PR DESCRIPTION
**Pull request by Montso-Matlaletsa**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+  #myMenu {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+  }
+  
+  /* Style the navigation links */
+  #myMenu li.hide { display: none; }
+  
+
```
Comment (Bandla1995) [inline css/style.css:96]: unnecessary space here

File: js/app.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+const filterStories = ()=>{
```
Comment (Bandla1995) [inline js/app.js:8]: i like this , i can use the javascript to add to to the script finding method.

File: js/app.js [hunk 3]
```diff
@@ -12,4 +18,89 @@ const filterStories = ()=>{
       if (item.indexOf(search) == -1) {i.classList.add("hide"); } //add hide class to an item
       else {i.classList.remove("hide");} //remove hide class
     }
-}
\ No newline at end of file
+}
+
+
+const store = (event) =>{
```
Comment (Bandla1995) [inline js/app.js:24]: the storage method looks bulky , i think we can reduce its complexity

File: index.html [hunk 4]
```diff
@@ -20,53 +20,29 @@ <h4>Today, 18-01-2022</h4>
         <div class="flex-grid">
             <div class="col">
                 <h2>Add Story</h2>
-                <form action="/action_page.php">
```
Comment (Bandla1995) [inline index.html]: i have never seen form action like this, i would like an explanation of how php was used here

Comment (Montso-Matlaletsa) [reply index.html]: this is deleted in a new pull request

File: js/app.js [hunk 5]
```diff
@@ -12,4 +18,89 @@ const filterStories = ()=>{
...
+    event.preventDefault();
+    let title = document.querySelector('#title').value;
```
Comment (Bandla1995) [inline js/app.js:26]: i like that you used queryselectors, instead of getElementById

**Pull request merged**